tests/scripts/thread-cert/border_router/test_vicarious_router_solicit.py

```diff
@@ -82,13 +84,15 @@ class MultiThreadNetworks(thread_cert.TestCase):
             'version': '1.2',
             'channel': CHANNEL2,
             'router_selection_jitter': 1,
+            'extended_panid': '08090a0b0c0d0e0f'
         },
         ROUTER2: {
             'name': 'Router_2',
             'allowlist': [BR2],
             'version': '1.2',
             'channel': CHANNEL2,
             'router_selection_jitter': 1,
+            'extended_panid': '08090a0b0c0d0e0f'
```

Comment:
Is it required for this test to use a specified extpanid ?

Reply:
Test cases in this file are assuming different on-link prefixes in the two Thread networks. I think some assertions could fail if the same extended pan id is used.

Reply:
SGTM. 

I guess the current on-link prefix mechanism works even if two different Thread networks are sharing the same extended panid. The on-link prefix would be same no matter which BR is advertising it.

Reply:
Yeah, just the test assertions are assuming different prefixes.